package.json

```diff
@@ -25,21 +33,25 @@
     "type": "git"
   },
   "devDependencies": {
-    "babel-eslint": "10.1.0",
-    "eslint": "7.17.0",
+    "@babel/core": "7.13.15",
+    "@babel/eslint-parser": "7.13.14",
+    "@babel/plugin-proposal-class-properties": "7.13.0",
+    "@parcel/babel-plugin-transform-runtime": "2.0.0-nightly.1823",
+    "eslint": "7.24.0",
     "eslint-config-semistandard": "15.0.1",
     "eslint-config-standard": "16.0.2",
     "eslint-plugin-import": "2.22.1",
     "eslint-plugin-node": "11.1.0",
-    "eslint-plugin-promise": "4.2.1",
+    "eslint-plugin-promise": "5.1.0",
+    "eslint-plugin-react": "7.23.2",
     "eslint-plugin-standard": "5.0.0",
-    "standard-version": "9.1.0",
-    "static-server": "*"
+    "parcel": "2.0.0-nightly.636",
+    "standard-version": "9.2.0"
   },
   "scripts": {
-    "deploy": "npm test && ssh broofa.com \"cd npm.broofa.com && ./deploy.sh\"",
-    "start": "static-server",
-    "open": "open -a \"Google Chrome\" http://localhost:9080",
+    "start": "rm -fr .parcel-cache && parcel index.html --dist-dir docs",
```

Comment:
There’s a `no-cache` flag that can be used on both the build and the server

Reply:
Does it do anything?  When I do `parcel serve --no-cache index.html`, it still generates the .parcel-cache directory.  😕